src/main/java/school/lemon/changerequest/java/introduction/pr1/Bob.java

```diff
@@ -1,10 +1,26 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Bob {
 
     public static String hey(String phrase) {
-        return "";
+
+
+        if (phrase.equals(phrase.toUpperCase()) ) {
```

Comment:
The thing is, that `"1 2 3".toUpperCase()` is the same as `1 2 3`. 
There are two ways, first one - use regexp figure out whether string contains upper case letters. Second one - additionally use `toLowerCase()`.